controllers/src/main/scala/cromwell/pipeline/controller/AuthController.scala

```diff
@@ -13,7 +13,8 @@ import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._
 import scala.concurrent.ExecutionContext
 import scala.util.Success
 
-class AuthController(authService: AuthService)(implicit executionContext: ExecutionContext) {
+class AuthController(authService: AuthService)(
+  implicit executionContext: ExecutionContext) {
```

Comment:
These lines are looks strange for me, as well.

Reply:
've changed